### PR TITLE
Disable DRTM in Zaius + Witherspoon

### PR DESCRIPTION
**Pull request by bofferdn**
Introducing DRTM in Hostboot tree requires it to be disabled in any config where SECUREBOOT/TPMDD is disabled

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/888)

### CODE REVIEW

**Comment by bofferdn** [comment]
Supersedes #PR886

**Comment by bofferdn** [comment]
This is a pre-req for Hostboot gerrit 35633 to pass op-build CI

**Comment by bofferdn** [comment]
garrison failed w/  "Requested sensor is not present" but the change does not impact so ... will retry it

File: openpower/configs/hostboot/zaius.config [hunk 1]
```diff
@@ -68,3 +68,5 @@ set CONSOLE_OUTPUT_FFDCDISPLAY
 unset SECUREBOOT
 unset TPMDD
 unset TPM_NUVOTON
+unset DRTM
```
Comment (williamspatrick) [inline openpower/configs/hostboot/zaius.config]: I don't see this config flag in the Hostboot code to make the comment, but why isn't this config:

    `default y if (SECUREBOOT && TPMDD)`
and/or
    `depends on SECUREBOOT && TPMDD`

**Comment by williamspatrick** [comment]
@bofferdn : I had a question two days ago on how this config variable is defined.  I don't think these are necessary if the HBconfig file is written correctly.

**Comment by bofferdn** [comment]
closing .. yep sorry .. made the suggested changes in op-build.  Closing this PR.

**Pull request closed without merging**